classes/class-wc-rest-connect-shipping-label-controller.php

```diff
@@ -24,6 +24,9 @@ public function __construct( WC_Connect_API_Client $api_client, WC_Connect_Servi
 	public function get( $request ) {
 		$order_id = $request[ 'order_id' ];
 		$payload = $this->shipping_label->get_label_payload( $order_id );
+		if ( ! $payload ) {
+			return new WP_Error( 404 );
```

Comment:
I don't believe this is the correct form for a 404. See the example here: https://developer.wordpress.org/rest-api/extending-the-rest-api/adding-custom-endpoints/#return-value